python/src/ignition/gazebo/event_manager.hh

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IGNITION_GAZEBO_PYTHON__EVENT_MANAGER_HPP_
+#define IGNITION_GAZEBO_PYTHON__EVENT_MANAGER_HPP_
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
```

Comment:
I have mixed feelings about using this shorthand. We may have other python interfaces in the codebase, like SWIG, so it may be good to be explicit. Also I think `pybind11` is short enough to spell out.